src/lighteval/utils.py

```diff
@@ -145,6 +145,22 @@ def flatten(item: list[Union[list, str]]) -> list[str]:
     return flat_item
 
 
+def boolstring_to_bool(x: Union[str, bool, int]) -> Union[bool, None]:
+    """Allows to manage string or bool to bool conversion, in case a configuration input is badly formatted.
+
+    Args:
+        x (str): A string (true, false, True, False, ...)
+
+    Returns:
+        Union[bool, None]: the corresponding boolean
+    """
+    if x in ["True", "true", True, 1]:
+        return True
+    if x in ["False", "false", False, 0]:
+        return False
+    return None
```

Comment:
Well `config["generation"]["multichoice_continuations_start_space"]` can be True/False/None so I wanted to cover all three

Reply:
well in that case we could add a third check like so.
```
    if x is None:
        return None
    Raise error
```

so that when user input something not expected, it fails.

Reply:
tell me what you think!

Reply:
good !